viewer/reconstruction.html

```diff
@@ -1112,6 +1126,33 @@
 
             function onDocumentMouseDown(event) {
                 window.focus();
+
+                intersected = raycaster.intersectObjects(point_clouds);
+                if (
+                    intersected.length > 0 &&
+                    options.drawLineOnPointClick && 
+                    selectedCamera
+                ){
+                    var material = new THREE.LineBasicMaterial( { color: 0xFFC0CB } );
+                    var geometry = new THREE.Geometry();
+                    geometry.vertices.push(new THREE.Vector3( 
+                        selectedCamera.geometry.boundingSphere.center.x,
+                        selectedCamera.geometry.boundingSphere.center.y,
+                        selectedCamera.geometry.boundingSphere.center.z,
+                    ));
+                    geometry.vertices.push(new THREE.Vector3( 
```

Comment:
similarly, this is using the intersection point (which depends on the position of the mouse over the square representing the point)  but it would be better to use the coordinates of the point that have been selected